### PR TITLE
Insert product details

### PR DESCRIPTION
**Pull request by shivanshu2394**

### CODE REVIEW

File: TrainingProject/TrainingProject/App_Start/RouteConfig.cs [hunk 1]
```diff
@@ -16,7 +16,7 @@ public static void RegisterRoutes(RouteCollection routes)
             routes.MapRoute(
                 name: "Default",
                 url: "{controller}/{action}/{id}",
-                defaults: new { controller = "Home", action = "Index", id = UrlParameter.Optional }
+                defaults: new { controller = "Home", action = "Products_Index", id = UrlParameter.Optional }
             );
```
Comment (salilsaini868) [inline TrainingProject/TrainingProject/App_Start/RouteConfig.cs]: Default route should be login

File: TrainingProject/TrainingProject/Controllers/HomeController.cs [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+        public HomeController()
+        {
+            strconnect = @"Data Source=172.20.21.129;MultipleActiveResultSets=True;Initial Catalog=RHPM;User ID=RHPM;Password=evry@123";
+        }
+
+        public int Product_ID { get; private set; }
+
+        // GET: Home
+        public ActionResult Products_Index()
```
Comment (salilsaini868) [inline TrainingProject/TrainingProject/Controllers/HomeController.cs]: Move your code to new product controller; remove from home controller

File: TrainingProject/TrainingProject/Models/HomeModel.cs [hunk 3]
```diff
@@ -0,0 +1,29 @@
+﻿
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace TrainingProject.Models
+{
+    public class HomeModel
```
Comment (salilsaini868) [inline TrainingProject/TrainingProject/Models/HomeModel.cs]: Rename to product model

File: TrainingProject/TrainingProject/Scripts/Products_AddPage.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
+﻿
```
Comment (salilsaini868) [inline TrainingProject/TrainingProject/Scripts/Products_AddPage.js]: rename js file

File: TrainingProject/TrainingProject/Views/Home/Product_Details.cshtml [hunk 5]
```diff
@@ -0,0 +1,163 @@
+﻿@*@model TrainingProject.Models.HomeModel
+
+@{
+    ViewBag.Title = "Product";
+}
+
+<style>
```
Comment (salilsaini868) [inline TrainingProject/TrainingProject/Views/Home/Product_Details.cshtml]: move css files

File: TrainingProject/TrainingProject/Views/Home/Product_Details.cshtml [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+
+</style>
+
+<h1>PRODUCTS</h1>
+<br />
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+
+        <script type="text/javascript">
+        $(document).ready(function () {
+            $('#form1').submit(function (e) {
```
Comment (salilsaini868) [inline TrainingProject/TrainingProject/Views/Home/Product_Details.cshtml]: duplicate js code ????

File: TrainingProject/TrainingProject/App_Start/RouteConfig.cs (hunk 1 above)
Comment (shivanshu2394) [reply TrainingProject/TrainingProject/App_Start/RouteConfig.cs]: Done Sir.

File: TrainingProject/TrainingProject/Controllers/HomeController.cs (hunk 2 above)
Comment (shivanshu2394) [reply TrainingProject/TrainingProject/Controllers/HomeController.cs]: Done Sir

File: TrainingProject/TrainingProject/Models/HomeModel.cs (hunk 3 above)
Comment (shivanshu2394) [reply TrainingProject/TrainingProject/Models/HomeModel.cs]: Done Sir

File: TrainingProject/TrainingProject/Scripts/Products_AddPage.js (hunk 4 above)
Comment (shivanshu2394) [reply TrainingProject/TrainingProject/Scripts/Products_AddPage.js]: Done Sir

File: TrainingProject/TrainingProject/Views/Home/Product_Details.cshtml (hunk 6 above)
Comment (shivanshu2394) [reply TrainingProject/TrainingProject/Views/Home/Product_Details.cshtml]: Done Sir

File: TrainingProject/TrainingProject/Views/Home/Product_Details.cshtml (hunk 5 above)
Comment (shivanshu2394) [reply TrainingProject/TrainingProject/Views/Home/Product_Details.cshtml]: Done Sir

**Pull request merged**